src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -104,6 +107,7 @@ import { NbChatMessageFile } from './chat-message-file.component';
 })
 export class NbChatMessageComponent {
 
+  @ContentChildren(NbChatCustomMessageDirective) customMessages: QueryList<NbChatCustomMessageDirective>;
```

Comment:
Sorry, I just noticed that we project custom templates into the chat message component. As we discussed that would produce template directives for every message which is unnecessary. Let's change it, so we project custom message templates into the chat component instead.